app/lib/dfe/bigquery/table.rb

```diff
@@ -33,11 +33,26 @@ def to_sql
         end
 
         if @order_clause.present?
-          base_sql << "ORDER BY #{@order_clause.keys.first} #{@order_clause.values.first.to_s.upcase}\n"
+          order_clause_key = @order_clause.keys.first
+          base_sql << "#{default_order_clause(order_clause_key)}, #{order_clause_key} #{@order_clause.values.first.to_s.upcase}\n"
         end
 
         base_sql
       end
+
+    private
+
+      def default_order_clause(order_clause_key)
+        <<~SQL
+          ORDER BY (
+            CASE WHEN #{order_clause_key}='Prefer not to say' THEN 4
+                 WHEN #{order_clause_key}='Unknown' THEN 3
+                 WHEN #{order_clause_key}='Other' OR #{order_clause_key}='Others' THEN 2
+                 ELSE 1
+            END
+          )
+        SQL
```

Comment:
I am not sure about this method being on the table in case we add another tables in the future. 🤔 
But for Monday it is okay.

Reply:
I know what you mean, but it was already dealing with the order clause here, so I followed through. Technically, it might work fine for other tables as well, but we'll figure it out when the time comes. Another approach I was thinking of was to do the sorting when presenting the data, but this felt more optimised.